WordPressAuthenticator/Signup/SignupEmailViewController.swift

```diff
@@ -140,7 +140,7 @@ class SignupEmailViewController: LoginViewController, NUXKeyboardResponder {
 
     private func checkEmailAvailability(completion:@escaping (Bool) -> ()) {
 
-        let remote = AccountServiceRemoteREST(wordPressComRestApi: WordPressComRestApi())
+        let remote = AccountServiceRemoteREST(wordPressComRestApi: WordPressComRestApi(baseUrlString: WordPressAuthenticator.shared.configuration.wpcomAPIBaseURL))
```

Comment:
I don't think we enforce it on this repo, but this line is rather long – could probably be broken up into smaller statements to be a bit more understandable?

Up to you :)